hw/ip/rstmgr/data/rstmgr.hjson.tpl

```diff
@@ -247,7 +233,7 @@
         },
 
         // reset requests include escalation reset + peripheral requests
-        { bits: "${4 + total_hw_resets - 1}:4",
+        { bits: "${3 + total_hw_resets - 1}:3",
```

Comment:
The three is for escalation, main power glitch, and sw reset (via rstmgr). Update the comment above?

Reply:
actually this was intentionally left this way.. because i didn't want to make too many assumptions inside `rstmgr` about what "hw_req" meant. The escalation one I specifically called out because it's different. I should also add `ndm_reset` here because it's also different, but the rest are meant to be somewhat "generic" hardware requests.

Reply:
actually main power glitch is special too..let me add that.

Reply:
updated.